CHANGELOG.md

```diff
@@ -3,16 +3,17 @@
 **This project is in Release Candidate stage.**
 
 ### Enhancements
-* None
+* Improve performance of interprocess mutexes on iOS which don’t need to support reader-writer locking. The primary beneficiary of this is beginning and ending read transactions, which is now almost as fast as pre-v13.0.0 (Core upgrade).
```

Comment:
From previous experience I believe this should go under fixed. We are not enhancing the dart sdk. See discussion for upgrade to v13.3.0

Reply:
Comparing to the [core changelog](https://github.com/realm/realm-core/blob/master/CHANGELOG.md) it is enhancement. Are you sure we have to move it under Fixes?

Reply:
The difference is very subtle but having this in Fixes is a tid better imo. Previously we decided to move it to fixes